# calib_refinement_icp
bag_file:
 - "/home/jjiao/catkin_ws/src/localization/LeGO-LOAM/LeGO-LOAM/data/yellow_20190104/8_shape/odom_top_localmap.bag"
 - "/home/jjiao/catkin_ws/src/localization/LeGO-LOAM/LeGO-LOAM/data/yellow_20190104/8_shape/odom_front_localmap.bag"
 - "/home/jjiao/catkin_ws/src/localization/LeGO-LOAM/LeGO-LOAM/data/yellow_20190104/8_shape/odom_tail_localmap.bag"
 
# all tf is at lidar frame 
##################################
tf_l1_l2_ini:
 #- [0.9982, -0.0436, 0.0406, 0]
 #- [0.0435, 0.9990, 0.0037, 0]
 #- [-0.0408, -0.0019, 0.9992, -1.0054]
 #- [0, 0, 0, 1.0000]
 - [0.9982, -0.0436, 0.0406, -1.952]
 - [0.0435, 0.9990, 0.0037, -0.402]
 - [-0.0408, -0.0019, 0.9992, -1.0054]
 - [0, 0, 0, 1.0000]
 #- [0.9982, -0.0436, 0.0406, 0.4204]
 #- [0.0435, 0.9990, 0.0037, -0.0007]
 #- [-0.0408, -0.0019, 0.9992, -1.2597]
 #- [0, 0, 0, 1.0000] 
 
tf_l1_l2_gt:
 - [0.9928, -0.0332, 0.0811, 0.4204]
 - [0.0349, 0.9988, -0.0086, -0.0007]
 - [-0.0804, 0.0138, 0.9938, -1.2597]
 - [0, 0, 0, 1.0000]

##################################
tf_l1_l3_ini:
 #- [1.0000, 0, 0, 0]
 #- [0, 1.0000, 0, 0]
 #- [0, 0, 1.0000, 0]
 #- [0, 0, 0, 1.0000]
 - [-0.9994, 0.0316, -0.0122, -1.8911]
 - [-0.0314, -0.9993, -0.0200, -0.0031]
 - [-0.0128, -0.0196, 0.9997, 0]
 - [0, 0, 0, 1.0000]
 
tf_l1_l3_gt:
 - [-0.9969, 0.0298, -0.0045, -1.9873]
 - [-0.0297, -0.9967, -0.0190, 0.0635]
 - [-0.0050, -0.0188, 0.9998, -1.1817]
 - [0, 0, 0, 1.0000]
 
################################## 
tf_l2_l3_ini:
 - [-0.9989,    0.0442,   -0.0180,   -2.0]
 - [-0.0437,   -0.9987,   -0.0264,   0.1474]
 - [-0.0192,   -0.0256,    0.9995,   -0.1185]
 - [0,         0,         0,    1.0000] 
 #- [-0.9989,    0.0442,   -0.0180,   -1.3691]
 #- [-0.0437,   -0.9987,   -0.0264,   -0.1734]
 #- [-0.0192,   -0.0256,    0.9995,   -0.1185]
 #- [0,         0,         0,    1.0000] 
 #- [-0.9963,   -0.0039,   -0.0858,   -2.4104]
 #- [0.0044,   1.0000,   -0.0050,     0.1474]
 #- [-0.0858,   -0.0054,    0.9963,   -0.1185]
 #- [0,         0,         0,    1.0000]
 
tf_l2_l3_gt:
 - [-0.9963,   -0.0039,   -0.0858,   -2.4104]
 - [0.0044,   1.0000,   -0.0050,     0.1474]
 - [-0.0858,   -0.0054,    0.9963,   -0.1185]
 - [0,         0,         0,    1.0000]
 
 
 
 
 
 
 
 
 
 
 
 